internal/gatewayapi/listener.go

```diff
@@ -227,16 +250,28 @@ func processAccessLog(envoyproxy *egv1a1.EnvoyProxy) *ir.AccessLog {
 
 				// TODO: remove support for Host/Port in v1.2
 				al := &ir.OpenTelemetryAccessLog{
-					Port:      uint32(sink.OpenTelemetry.Port),
 					Resources: sink.OpenTelemetry.Resources,
 				}
 
-				if sink.OpenTelemetry.Host != nil {
-					al.Host = *sink.OpenTelemetry.Host
+				// TODO: how to get authority from the backendRefs?
+				ds, err := t.processBackendRefs(sink.OpenTelemetry.BackendRefs, envoyproxy.Namespace, resources)
+				if err != nil {
+					return nil, err
+				}
+				al.Destination = ir.RouteDestination{
+					Name:     fmt.Sprintf("accesslog-%d", idx), // TODO: rename this, so that we can share backend with tracing?
```

Comment:
+1 to this prefix, once we add cluster level features like `circuitBreaking` which will live on `backendRef` level , we won't be able to reuse `tracing-` here

Reply:
but we should reuse with tracing provider, normally there's only one collector per cluster.

Reply:
it may not be possible to reuse if we decide to put features inside `backendRef` field, if feature live in top level policy, it may be possible